command/v7/actor.go

```diff
@@ -38,6 +38,7 @@ type Actor interface {
 	CreateDockerPackageByApplication(appGUID string, dockerImageCredentials v7action.DockerImageCredentials) (v7action.Package, v7action.Warnings, error)
 	CreateDockerPackageByApplicationNameAndSpace(appName string, spaceGUID string, dockerImageCredentials v7action.DockerImageCredentials) (v7action.Package, v7action.Warnings, error)
 	CreateIsolationSegmentByName(isolationSegment v7action.IsolationSegment) (v7action.Warnings, error)
+	CreateManagedServiceInstance(serviceOfferingName, servicePlanName, serviceInstanceName, serviceBrokerName, spaceGUID string) (v7action.Warnings, error)
```

Comment:
That's quite a complex signature. Some other resources have opted for passing an object such as `CreateApplicationInSpace` and `CreateIsolationSegmentByName`.

Reply:
Yes, it became pretty annoying specially now that I am adding also tags and parameters, so doing that refactor within the tags story.